### ISSUE TITLE
Missing red arc in fuel level indicators

### ISSUE THREAD

**Issue by MariuszXC** [c172p-team/c172p#1386]
In a document titled 1959 Cessna 172 Manual.pdf I found on the net, in section IV - Operating limitations, there is a statement on page 4-2:

Fuel Quantity Indicators.
  Empty (2-1/2 gallons unusable each tank) ...  E (red line)
  Not recommended for take-off ... E to 1/4 (red arc).

The descriptions clearly reference a red line and red arc on the fuel gauge scale. 

Similar reference is made on page 1-4 under "Fuel Quantity Indicators" section.

I don't know if these markings were carried over to later years models, but I don't see a reason why not, so maybe this missing bit should be added to fuel gauges model?

The firt page of the PDF I referenced is titled 1959 Cessna 172 Owners Manual N7084T

**Comment by wlbragg** [comment]
Thank you MariuszXC, I'll try to take a look at this potential addition in the next few days.

**Comment by MariuszXC** [comment]
Thank you for taking a note. I would love to send a pull request instead, but I am afraid I do not have necessary blender skills, and that is the area I think is needed in this case..

**Comment by wlbragg** [comment]
@dany93 

I'm planning on trying to clean up some of the outstanding issues. Please don't feel obligated to participate in reviewing my work, but if you don't mind, it would be nice to continue forward as we have been.

**Comment by dany93** [comment]
@wlbragg 
OK for the red line.
Do you you intend trying to add a red arc (E to 1/4) or is it enough?

**Comment by wlbragg** [comment]
>Empty (2-1/2 gallons unusable each tank) ... E (red line)
Not recommended for take-off ... E to 1/4 (red arc).

I searched google images for c172p fuel gauge. I couldn't find one with a red arch. Also, "arch" depicts a different type of fuel gauge, a gauge with a semicircle scale where the red arch is part of the line with the tick marks (bottom image). I like the idea of the "red line" for safety reasons.
Our gauge is a straight scale. I decided on the current one because it is similar to our original one but it contains more information, including the red "line". it is from a picture of what looks like a real aircraft that is very close to our model.

![fgauge](https://github.com/c172p-team/c172p/assets/11069680/1381140d-4cee-4238-b359-ffb2af0ad11a)

So I guess a compromise?
